### PR TITLE
Default workflow archiving module

### PR DESCRIPTION
**Pull request by pavelhalabala**
Implements issue https://github.com/Netflix/conductor/issues/1563

### CODE REVIEW

File: contribs/src/test/java/com/netflix/conductor/contribs/listener/ArchivingWorkflowStatusListenerTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package com.netflix.conductor.contribs.listener;
```
Comment (apanicker-nflx) [inline contribs/src/test/java/com/netflix/conductor/contribs/listener/ArchivingWorkflowStatusListenerTest.java]: Please add license header to the file.

Comment (pavelhalabala) [reply contribs/src/test/java/com/netflix/conductor/contribs/listener/ArchivingWorkflowStatusListenerTest.java]: Done

Comment (pavelhalabala) [reply contribs/src/test/java/com/netflix/conductor/contribs/listener/ArchivingWorkflowStatusListenerTest.java]: Also extended documentation here: https://github.com/Netflix/conductor/pull/1565/files#diff-137287826e42d8055e71af8bf4e66e8b

**Comment by pavelhalabala** [comment]
Review comments were applied

**Pull request merged**